src/View/formation/index.html.twig

```diff
@@ -0,0 +1,70 @@
+{% extends 'layout.html.twig' %}
+{% block stylesheet %}<link rel="stylesheet" href="/assets/css/formation.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/remixicon/3.4.0/remixicon.css" crossorigin="">
```

Comment:
this link is already imported in layout remove it and import parent content inside your stylesheet block